packages/react-sdk/src/core/components/StreamCall/StreamCall.tsx

```diff
@@ -1,4 +1,12 @@
-import { StreamCallProvider } from '@stream-io/video-react-bindings';
+import { ComponentType, PropsWithChildren } from 'react';
+import {
+  StreamCallProvider,
+  StreamCallProviderProps,
+} from '@stream-io/video-react-bindings';
 
 // re-exporting the StreamCallProvider as StreamCall
-export const StreamCall = StreamCallProvider;
+export const StreamCall: ComponentType<
+  PropsWithChildren<StreamCallProviderProps>
+> = StreamCallProvider;
+
+StreamCall.displayName = 'StreamCall';
```

Comment:
I have mixed feelings about whether this should go to `react-bindings` or not. Any thoughts?